tests/pathfinding/test_fee_schedule.py

```diff
@@ -131,6 +131,8 @@ def test_fees_in_routing():
     # Now let's try imbalance fees
     tn.set_fee(2, 3, imbalance_penalty=[(TA(0), FA(0)), (TA(200), FA(200))])
     assert tn.estimate_fee(1, 3) == 10
+
+    tn.set_fee(2, 3, imbalance_penalty=[(TA(0), FA(200)), (TA(200), FA(0))])
     assert tn.estimate_fee(3, 1) == -10
```

Comment:
Why do you flip both the `imbalance_penalty` and the transfer direction at the same time? This makes it harder to compare it with the assert above.

Why not do
```python
# Now let's try imbalance fees
tn.set_fee(2, 3, imbalance_penalty=[(TA(0), FA(0)), (TA(200), FA(200))])
assert tn.estimate_fee(1, 3) == 10
assert tn.estimate_fee(3, 1) == -10
```
?

I'm also surprised that your code passes. After flipping both direction and fee schedule, I would have expected the same fee as before. Where is the error in my reasoning?

Reply:
This is weird, the test passes like this as well:
```py
    tn.set_fee(2, 3, imbalance_penalty=[(TA(0), FA(0)), (TA(200), FA(200))])
    assert tn.estimate_fee(1, 3) == 10
    assert tn.estimate_fee(3, 1) == 10

    tn.set_fee(2, 3, imbalance_penalty=[(TA(0), FA(200)), (TA(200), FA(0))])
    assert tn.estimate_fee(1, 3) == -10
    assert tn.estimate_fee(3, 1) == -10
```